concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountAddress.java

```diff
@@ -32,6 +38,25 @@ public static AccountAddress from(String address) {
         return AccountAddress.from(addressBytes);
     }
 
+    public AccountAddress newAlias(int alias) {
+        if (alias < 0) {
+            throw new NumberFormatException("Alias must be non negative");
+        }
+        if (alias > ALIAS_MAX_VALUE) {
+            alias = alias % ALIAS_MAX_VALUE;
+        }
+        val buffer = ByteBuffer.allocate(Integer.BYTES);
+        buffer.putInt(alias);
```

Comment:
Is this big-endian?

Reply:
It should ideally be big endian, since that is what we do in other SDKs.

Reply:
Yes big endian. https://docs.oracle.com/javase/7/docs/api/java/nio/ByteBuffer.html